recipes/brb-seqtools/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.1" %}
+{% set name = "BRB-seqTools" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+build:
+  number: 0
```

Comment:
add `noarch: python`

Reply:
Yes, ignore the comment. We don't do `noarch` yet. I was confused with conda-forge requirements. There, "universal" python packages, i.e. pure python compatible with all versions, are built with `noarch` as they don't need selectors and work with current `conda-build` already.